### DISCUSSION TITLE
私密文件夹密码保护

### DISCUSSION THREAD

**Discussion by lyizhuo** [spencerwooo/onedrive-vercel-index#417] [Q&A] [answered]
私密文件夹不需要输入密码就能打开查看文件名称，只是打开私密文件夹内的文件时才需要输入密码？

**Comment by lyizhuo** [comment, marked as answer]
已解决

**Comment by spencerwooo** [comment]
不是，比如这个文件夹：https://drive.spencerwoo.com/🌞%20Private%20folder/u-need-a-password

文件夹是 protected，查看文件夹里的内容就需要密码。

**Reply by lyizhuo** [reply]
不知道出现了什么问题，重新保存site.config以后就正常了
